tests/test_dataset.py

```diff
@@ -35,12 +37,19 @@
     InterpColumnAlignConfig,
 )
 
+if TYPE_CHECKING:
+    from rbyte.batch import Batch
+
 logger = get_logger(__name__)
 
 CONFIG_PATH = "../config"
 DATA_DIR = Path(__file__).resolve().parent / "data"
 
 
+def norm(x: Tensor, **kwargs: Any) -> Tensor:  # noqa: ANN401
+    return torch.linalg.norm(x, **kwargs)  # pyright: ignore[reportUnknownVariableType, reportUnknownMemberType]
```

Comment:
basedpyright complains about all the outputs of `torch.linalg.norm`down the road so i've made this wrapper. pretty sure it's not the best way to do it tho, open to discussions

Reply:
i'd say use `torch.linalg.norm` as is and `# pyright: ignore` where necessary